modules/openapi-generator/src/main/resources/swift5/modelOneOf.mustache

```diff
@@ -1,4 +1,4 @@
-public enum {{classname}}: {{#useVapor}}Content{{/useVapor}}{{^useVapor}}Codable{{/useVapor}} {
+public enum {{classname}}: {{#useVapor}}Content{{/useVapor}}{{^useVapor}}Codable, Hashable{{/useVapor}} {
```

Comment:
@denizdogan can you please replace `, Hashable` with `{{#vendorExtensions.x-swift-hashable}}, Hashable{{/vendorExtensions.x-swift-hashable}}`?
So that this Hashable implementation respects the rules created in https://github.com/OpenAPITools/openapi-generator/pull/9495.
Thanks

Reply:
Should be fixed now! Thanks